### ISSUE TITLE
TinyGo support

### ISSUE THREAD

**Issue by kyleconroy** [planetscale/vtprotobuf#31]
First off, wanted to say thanks for such a great project. While the issue title may sound like a feature request, the code generated by vtprotobuf already works with TinyGo. You can see an example of that here: https://github.com/kyleconroy/go-wasm-plugins

What I'd like to ask about is making TinyGo support explicit via a CI job. Is this something you'd be interested in supporting? If so I can take a first pass.

**Comment by vmg** [comment]
Yeah for sure, I wouldn't mind running a TinyGo CI build (it it doesn't take a prohibitively long amount of time to run -- which it shouldn't!). Please feel free to send a Pull Request. Thanks!
